ProjectCars/CarCRUD.cs

```diff
@@ -1,27 +1,25 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using System.IO;
 using System.Linq;
-using System.Text;
-using System.Text.Json;
-using System.Threading.Tasks;
+using Newtonsoft.Json;
 
 namespace ProjectCars
 {
-     public class CarCRUD
+    public class CarCRUD
     {
-        //private static string path = @"C:\Json Store\Cars.json";
+        private static string _path = @"C:\Json Store\Cars.json"; //I´m going to change this
 
         public static List<Car> GetCars()
         {
-            var cars = new List<Car>
+            List<Car> cars = new List<Car>
```

Comment:
This method is in work progress, right? This should return all the cars in the json File?

Reply:
Yes, you are right. I´m not finished with that.